README.md

```diff
@@ -24,124 +24,24 @@ What can you do with Effortless Infrastructure?
 
 - [Chef Automate](https://github.com/chef/automate) - Enterprise dashboard and analytics tool enabling cross-team collaboration with actionable insights for configuration and compliance and an auditable history of changes to environments.
 
-## Existing Users - Effortless for Chef Infra
+## Existing Users
 
-If you're already familiar with Chef Infra, here's a quick rundown of how Effortless for Chef Infra works.
+If you're already familiar with the Chef stack, here's a quick rundown of how Effortless works.
 
-1. Effortless uses a strong build process for your cookbooks. The build creates a single, deployable package with your cookbooks, an up-to-date Chef Infra Client, and the latest best practices.
+1. Effortless uses a build process to pull down all your cookbooks or profiles. The build creates a single, deployable package. For Chef Infra, it contains your cookbooks, an up-to-date Chef Infra client, and the latest best practices. For Chef InSpec, it contains your profiles, an up-to-date Chef InSpec client, and the latest best practices.
 
 2. At runtime, Chef Infra works without Chef Infra Server. It uses Chef Solo mode.
 
-3. Chef Habitat manages Chef Infra, and provides a pull-based update strategy for continuous delivery of the Chef Infra Client and your cookbooks.
+3. At runtime, Chef InSpec works without pulling profiles from Chef Automate. All profiles, including those from Chef Automate, are vendored at build time.
+
+3. Chef Habitat manages Chef Infra and Chef InSpec, and provides a pull-based update strategy for continuous delivery.
```

Comment:
Need to fix the numbering here.

Reply:
Doh. I fixed it forever using markdown's smarts.